.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest a new feature or other enhancement.
+labels: enhancement
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Could we link to an example of a good feature request to emphasize what "clear and concise" is?

Reply:
Doesn't have to be from this project, we could choose one of the providers.